### PR TITLE
Improve research participants section with how to

### PR DESCRIPTION
**Pull request by rrhyne**
Improvements to the handbook based on a question from @jyavorska.

### CODE REVIEW

**Comment by rrhyne** [comment]
@quinnkeast, you've probably sourced more customers for your testing than anyone. Could you take a few moments to add your thoughts here? Also, is the [product research](https://sourcegraph.com/users/rrhyne/settings/product-research) section a resource that should be called out here?

File: content/product/design/research/index.md [hunk 1]
```diff
@@ -44,9 +44,23 @@ After research is complete, we publish our findings in the research in the [user
 
 As our users are primarily developers, our own Sourcegraph developers are great interview participants. (However, we need to remember that our own developers are generally very experienced in the area of code search, which results in implicit bias that we must account for in synthesis.)
 
+Use the followign methods to identify internal research subjects:
```
Comment (marisak) [inline content/product/design/research/index.md]: Small typo in line 47, should be "following"

File: content/product/design/research/index.md [hunk 2]
```diff
@@ -44,9 +44,23 @@ After research is complete, we publish our findings in the research in the [user
...
+
+- Post a request in #dev-chat, #ce, #community-chat or #customer support with information about the area you are researching
+- Often, developers have little bandwith for new requests. Making direct requests of specific developers tends to yeild a higher response rate
```
Comment (marisak) [inline content/product/design/research/index.md]: Small typo in line 50, "yield"

File: content/product/design/research/index.md (hunk 1 above)
Comment (rrhyne) [reply content/product/design/research/index.md]:
```suggestion
Use the following methods to identify internal research subjects:
```

**Comment by rrhyne** [comment]
Thanks for spotting those typos @marisak!

**Review by marisak — APPROVED** [review]
Made my comments inline, looks great!

File: content/product/design/research/index.md [hunk 3]
```diff
@@ -44,9 +44,23 @@ After research is complete, we publish our findings in the research in the [user
...
+
+- Post a request in #dev-chat, #ce, #community-chat or #customer support with information about the area you are researching
+- Often, developers have little bandwith for new requests. Making direct requests of specific developers tends to yield a higher response rate
+
 ## External users
 
-Existing and prospective customers, participants we find through outreach on social media, and other sources are all great participants for research activities.
+Relying on internal users can lead to introducing bias into your research, yet sourcing external users for feedback is one of the most difficult parts of the research process. The following are methods product and design use to source research candidates:
+
+- Outreach on social media, either through our platform or via active and influential accounts
```
Comment (jjinnii) [inline content/product/design/research/index.md]: This is a pretty new concept for me (my past companies didn't actively use social media like Sourcegraph). Are there company accounts that we can access to request feedback? (i.e. what do we mean by active/influential accounts)

File: content/product/design/research/index.md [hunk 4]
```diff
@@ -44,9 +44,23 @@ After research is complete, we publish our findings in the research in the [user
...
+- The #feedback Slack channel can be a source of participants for specific issues
+  - Use Productboard to identify multiple feedback entries related to a single issue
+- Consult customer engineering or support to identify customers who may be interested or have feedback on the area of research. Additionally, the following methods are helpful when locating customers to interview:
+  - Searching Slack support channels for users interested in the product space
```
Comment (jjinnii) [inline content/product/design/research/index.md]: Does support channels mentioned in this point and below refer to # feedback or are there other CE support channels?

**Comment by jjinnii** [comment]
@rrhyne Looks great! Thanks so much for working on this and the user research directory, they're both super useful.

File: content/product/design/research/index.md (hunk 4 above)
Comment (rrhyne) [reply content/product/design/research/index.md]:
```suggestion
  - Searching Slack support channels (#support-{customer-name}) for users interested in the product space
```

Comment (rrhyne) [reply content/product/design/research/index.md]: @jjinnii does this clarify a bit?

Comment (jjinnii) [reply content/product/design/research/index.md]: @rrhyne Yes it does! I actually didn't know that's how those channels were named :)

File: content/product/design/research/index.md (hunk 3 above)
Comment (rrhyne) [reply content/product/design/research/index.md]:
```suggestion
- Outreach on social media, either through our platform (ask for a post in #social-media-action) or via active and influential accounts such as the CTO, CEO, or members of your team. 
```

Comment (rrhyne) [reply content/product/design/research/index.md]: Does this help?

Comment (jjinnii) [reply content/product/design/research/index.md]: @rrhyne Yes!

**Review by jasonyavorsky — APPROVED** [review]
Looks great!

**Pull request merged**